ncdap_test/tst_urls.sh

```diff
@@ -83,11 +83,8 @@ COLUMBIA="http://iridl.ldeo.columbia.edu/SOURCES/.Models/.NMME/.NASA-GMAO/.MONTH
 # Known to fail
 
 XFAILTESTS=
-
-# Suppress some tests if not windows platform.
-if test "x$FP_ISMSVC" != xyes ; then
-    XFAILTESTS="$XFAILTESTS test.67"
-fi
+# Suppress some tests
+XFAILTESTS="$XFAILTESTS test.67"
```

Comment:
Should this test just be removed if it's going to be universally suppressed? What does this test do? I'll dig into it for an answer myself, but reviewing this at the moment.

Reply:
I did not look to see why it failed.